core/uwsgi.c

```diff
@@ -1009,6 +1009,7 @@ static struct uwsgi_option uwsgi_base_options[] = {
 	{"config-py", no_argument, 0, "dump the uwsgiconfig.py used for building the core  (useful for building external plugins)", uwsgi_opt_config_py, NULL, UWSGI_OPT_IMMEDIATE},
 	{"build-plugin", required_argument, 0, "build a uWSGI plugin for the current binary", uwsgi_opt_build_plugin, NULL, UWSGI_OPT_IMMEDIATE},
 	{"version", no_argument, 0, "print uWSGI version", uwsgi_opt_print, UWSGI_VERSION, 0},
+	{"response-headers", required_argument, 0, "Set response head maximum size (Default is page size)", uwsgi_opt_set_int, &uwsgi.response_header_size, 0},
```

Comment:
This should probably be `--response-headers-buffer-size`, to be consistent with the other options.  You may also want to word it like the others, being `set response header maximum size (default: page size)`.